OpenConsole.sln

```diff
@@ -300,6 +300,10 @@ Project("{8BC9CEB8-8B4A-11D0-8D11-00A0C91BC942}") = "winconpty.Tests.Feature", "
 EndProject
 Project("{8BC9CEB8-8B4A-11D0-8D11-00A0C91BC942}") = "TerminalAzBridge", "src\cascadia\TerminalAzBridge\TerminalAzBridge.vcxproj", "{067F0A06-FCB7-472C-96E9-B03B54E8E18D}"
 EndProject
+Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "_dep", "_dep", "{6FFCC3B4-A6CB-44FF-BFE3-1EF02CCD7557}"

```

Comment:
I thought we already had a top-level "_Dependencies" folder in the solution?

Reply:
That one doesn’t seem to contain any buildable projects. I’m happy to move this in there, but /shrug!

Reply:
I'd rather you put it in the existing one, in this case.